### ISSUE TITLE
request: add windows/arm64 build for Windows on ARM

### ISSUE THREAD

**Issue by rubin55** [syncthing/syncthing#7445]
### Include required information
Really like its sister issue #7128, add windows/arm64 support and provide a download for Syncthing on Windows on ARM.

Go is very close to official build support and there's an official build that's also being used for Wireguard, see: 

https://github.com/golang/go/issues/36439
https://github.com/thongtech/go-windows-arm

**Comment by rubin55** [comment]
Just a note for when you guys get to this. You will run into two issues:

1. go-ole: https://github.com/go-ole/go-ole/issues/202
2. goversioninfo: https://github.com/golang/go/issues/38020

I was able to fix 1. by simply creating the variant and variant-date files for arm with the correct build-directive. I was not able to fix the error I got with goversion info (`unknown ARM relocation type 7`). It should be something obvious though. The golang issue does not provide a fix (it's about someone building on Linux on ARM, where goversioninfo seems to be invoked unintendedly).
